lint/tfsec.bzl

```diff
@@ -0,0 +1,88 @@
+"""API for declaring a tfsec lint aspect that visits filegroup rules.
+
+Typical usage:
+
+Use [tfsec_aspect](#tfsec_aspect) to declare the tfsec linter aspect, typically in in `tools/lint/linters.bzl`:
+
+```
+load("@aspect_rules_lint//lint:tfsec.bzl", "tfsec_aspect")
+
+tfsec = tfsec_aspect(
+    binary = "@multitool//tools/tfsec",
+)
+```
+"""
+
+load("//lint/private:lint_aspect.bzl", "LintOptionsInfo", "report_files", "should_visit")
+
+_MNEMONIC = "AspectRulesLintTfsec"
+
+def _tfsec_action(ctx, executable, srcs, stdout, exit_code = None):
+    args = ctx.actions.args()
+    args.add_all([
+        ctx.label.package,
+        "--exclude-downloaded-modules",
+        "--minimum-severity=HIGH",
```

Comment:
nit/future: should users be able to configure this somewhere? does tfsec not read any configuration file?

Reply:
Yeah, it has a config file. This was the MVP that I needed for Silo use.